internal/test/scorecard/tests.go

```diff
@@ -72,6 +76,61 @@ func CryostatCRTest(bundle *apimanifests.Bundle, namespace string, openShiftCert
 	return *r
 }
 
+func CryostatConfigChangeTest(bundle *apimanifests.Bundle, namespace string, openShiftCertManager bool) scapiv1alpha3.TestResult {
+	tr := newTestResources(CryostatConfigChangeTestName)
+	r := tr.TestResult
+
+	err := setupCRTestResources(tr, openShiftCertManager)
+	if err != nil {
+		return fail(*r, fmt.Sprintf("failed to set up %s test: %s", CryostatConfigChangeTestName, err.Error()))
+	}
+
+	// Create a default Cryostat CR with default empty dir
+	cr := newCryostatCR(CryostatConfigChangeTestName, namespace, !tr.OpenShift)
+	cr.Spec.StorageOptions = &operatorv1beta1.StorageConfiguration{
+		EmptyDir: &operatorv1beta1.EmptyDirConfig{
+			Enabled: true,
+		},
+	}
+
+	_, err = createAndWaitTillCryostatAvailable(cr, tr)
+	if err != nil {
+		return fail(*r, fmt.Sprintf("%s test failed: %s", CryostatConfigChangeTestName, err.Error()))
```

Comment:
Maybe, let's use a similar log message to other tests?

https://github.com/cryostatio/cryostat-operator/blob/cfcbfc74fab65b78bf5fb63a8c5abe658d65fe6a/internal/test/scorecard/tests.go#L88